iidm/iidm-api/src/main/java/com/powsybl/iidm/network/Injection.java

```diff
@@ -18,4 +21,27 @@
      */
     Terminal getTerminal();
 
+    default List<OperationalLimits> getOperationalLimits() {
+        return Collections.emptyList();
+    }
+
+    default <L extends OperationalLimits> L getOperationalLimits(LimitType type, Class<L> limitClazz) {
+        return null;
+    }
+
+    default CurrentLimitsAdder newCurrentLimits() {
+        throw new UnsupportedOperationException();
```

Comment:
Why do we have this exception ?

Reply:
It is only to prevent breaking change (error in implementation - for example in persistent IIDM)